src/EditorFeatures/Core/Implementation/KeywordHighlighting/HighlighterViewTaggerProvider.cs

```diff
@@ -99,8 +101,12 @@ protected override async Task ProduceTagsAsync(TaggerContext<KeywordHighlightTag
             {
                 var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
 
-                var spans = _highlightingService.GetHighlights(root, position, cancellationToken);
-                foreach (var span in spans)
+                using var _ = s_listPool.GetPooledObject();
+                var highlights = _.Object;
```

Comment:
@sharwell how would you feel about this sort of construct:


```c#
using var _ = s_listPool.GetPooledObject(out var highlights);
```
?

Reply:
I prefer this over the current form, fwiw. Having a variable named `_` is dubious to use.

Reply:
done.